ios/magnet/AppDelegate.m

```diff
@@ -11,6 +11,8 @@
 
 #import "RCTRootView.h"
 
+#import "magnet-Swift.h"
```

Comment:
Yes it's a bit tricky, I'll comment on it on the code too.

As we are using a class in swift in the app delegate (the notifications), in order to include swift code into objective-c one you need that 'automagically' generated header:
<projectname>-Swift.h